src/data/faq.json

```diff
@@ -341,7 +347,7 @@
                                 "anchor": "risk_update_24",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated March 30, 2022</b>",
+                                    "<div class='update'>The information in this FAQ entry is related to features or describes features of the app that are no longer available as of May 1, 2023, and is therefore no longer up-to-date. For more information, click here: <a href='#ramp_down' target='_blank'>What is the background for the end of the development of the CWA?</a><hr />The information in this FAQ entry is related to features or describes features of the app that are no longer available as of May 1, 2023, and is therefore no longer up-to-date. For more information, click here: <a href='#ramp_down' target='_blank'>What is the background for the end of the development of the CWA?</a></div>",

```

Comment:
This has been added twice.

Reply:
Removed the duplicate in https://github.com/corona-warn-app/cwa-website/pull/3474/commits/3420f6ef18b4aab32751f66db3cda39b99703297, thanks!